api/institutions/views.py

```diff
@@ -418,10 +416,22 @@ class InstitutionImpactList(JSONAPIBaseView, ListFilterMixin, generics.ListAPIVi
 
     view_category = 'institutions'
 
+    csv_export = False
+
+    def initiate_csv_export_settings(self):
+        self.pagination_class = MaxSizePagination
```

Comment:
question: could this be done by making `pagination_class` a property/getter?
```py
@property
def pagination_class(self):
    return MaxSizePagination if self.is_csv_export() else super().pagination_class
```
if so (and if my suggestion re: `is_csv_export` pans out), could get rid of `initiate_csv_export_settings`, which i think would be a readability win -- easier to understand why `pagination_class` is what it is when the logic is in the same place as the value